README.md

```diff
@@ -1,7 +1,7 @@
 # [PredictionIO](http://prediction.io)
 
 [![Build
-Status](https://travis-ci.org/PredictionIO/PredictionIO.svg?branch=develop)](https://travis-ci.org/PredictionIO/PredictionIO)
+Status](https://api.travis-ci.org/apache/incubator-predictionio.svg?branch=develop)](https://travis-ci.org/PredictionIO/PredictionIO)
```

Comment:
We should also change the following `https://travis-ci.org/PredictionIO/PredictionIO` into `https://travis-ci.org/apache/incubator-predictionio`.